src/main/java/seedu/address/logic/commands/schedule/EditScheduleCommand.java

```diff
@@ -53,7 +53,7 @@
 
     public static final String MESSAGE_EDIT_SCHEDULE_SUCCESS = "Schedule Edited: \n%1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
-    public static final String MESSAGE_DUPLICATE_SCHEDULE = "This Schedule overlaps with an existing Schedule";
+    public static final String MESSAGE_DUPLICATE_SCHEDULE = "This Schedule already exists in the Schedule List";
```

Comment:
to keep it consistent, would it be better to be
"The Schedule already exists in FitEgo"
and also in case they think Schedule List is the "today's schedule" in home page